### PR TITLE
Refactored logging import in Sample Network Tests

### PR DESCRIPTION
**Pull request by rewati-arista**
# Please include a summary of the changes

* sample_network_tests -> imports new logger, black formatting

# Include the Issue number and link

https://github.com/aristanetworks/vane/issues/517

# Type of change

- - [ ] Bug fix (non-breaking change which fixes an issue)
- - [ ] New feature (non-breaking change which adds functionality)
- - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- - [ ] This change requires a documentation update
- - [X] Other : Refactor

# Effort required on reviewers end

- - [X] Easy
- - [ ] Medium
- - [ ] Hard 

# How Has This Been Tested?
    
## New feature

Logs get created for test case
Screenshot 2023-09-08 at 8 56 36 AM
    
# CI pipeline result

- - [X] Pass
- - [ ] Fail

### CODE REVIEW

**Review by shachiagarwal — APPROVED** [review]

File: sample_network_tests/lldp/test_lldp.py [hunk 1]
```diff
@@ -249,8 +238,7 @@ def test_if_lldp_system_name_is_correct_on_(self, dut, tests_definitions):
         if tops.actual_output == tops.expected_output:
             tops.test_result = True
             tops.output_msg = (
-                f"{tops.dut_name}'s LLDP System Name is correct: "
-                f"{tops.expected_output}"
+                f"{tops.dut_name}'s LLDP System Name is correct: " f"{tops.expected_output}"
```
Comment (grybak-arista) [inline sample_network_tests/lldp/test_lldp.py:241]: Black combined 2 f-strings without actually combining the strings 😮

File: sample_network_tests/log/test_logging.py [hunk 2]
```diff
@@ -84,14 +85,11 @@ def test_if_log_messages_appear_on_(self, dut, tests_definitions):
...
-                    tops.output_msg += (
-                        f"{tops.dut_name} system logs contains message: {sys_msg}. "
-                    )
+                    tops.output_msg += f"{tops.dut_name} system logs contains message: {sys_msg}. "
                 else:
                     actual_output = False
                     tops.output_msg += (
-                        f"{tops.dut_name} system logs does NOT contains message: "
-                        f"{sys_msg}. "
+                        f"{tops.dut_name} system logs does NOT contains message: " f"{sys_msg}. "
```
Comment (grybak-arista) [inline sample_network_tests/log/test_logging.py:92]: Black didn't combine the strings correctly... again!

**Review by grybak-arista — APPROVED** [review]
Approved, but there are a couple of instances where Black formatting didn't combine the strings correctly, marked by the comments.

File: sample_network_tests/log/test_logging.py (hunk 2 above)
Comment (rewati-arista) [reply sample_network_tests/log/test_logging.py]: black disappointing us... again!

**Pull request merged**